modules/installation-supported-aws-machine-types.adoc

```diff
@@ -29,6 +29,11 @@ endif::[]
 
 The following Amazon Web Services (AWS) instance types are supported with {product-title}.
 
+[NOTE]
+====
+Use the machine types included in the following charts for your instances. If you use an instance type that is not listed in the chart, ensure that the instance size you use matches the standard performance of the listed instance type. 
```

Comment:
I am a bit confused about _standard performance of the listed instance type_,  what does _standard performance_ 
 mean here? 

Actually, we do have [Minimum resource requirements for cluster installation](https://kelbrown20.github.io/bug-fix-previews/BZ1996012-instance-type-clarification/installing/installing_aws/installing-aws-customizations.html#installation-minimum-resource-requirements_installing-aws-customizations) section, I think let user reference this table (minimum resource) is more straightforward.

Reply:
The _Standard performance_ is more of a generalization referencing this chart: https://aws.amazon.com/ec2/instance-types/
As in, if the user uses a type not listed in the chart in the OCP docs (but listed in the AWS docs), the performance has to match that of one listed in the OCP docs chart.

Does that help at all? Or do you think something need changing? 
Thank you!

Reply:
@kelbrown20 
I think the `minimum resource` description is very clear that users can choose instance types based on CPU cores and memory,  the `standard performance` is not a quantifiable indicator, e.g. is the CPU frequency part of `standard performance`? maybe yes, but OCP has no restriction on this.

Another question is, I saw https://github.com/openshift/openshift-docs/pull/49799 is back, does this PR conflict with it? instance type (e.g. `m5.2xlarge`) will be replaced by instance type family (e.g. `m5`)

Reply:
@yunjiang29 Hello! Thats a great point that `standard performance` is not quantifiable and can be confusing, I updated the Note using the `minimum resource requirement` wording instead. 
Regarding #49799, once that PR is merged I can update this one with the new modules. 

Since there are still AWS instance types that are not tested, but listed in the AWS documentation, do you think this NOTE could still be helpful for users that use types not listed in this OCP doc chart?

Reply:
@kelbrown20 thanks for the updates.

> Since there are still AWS instance types that are not tested, but listed in the AWS documentation, do you think this NOTE could still be helpful for users that use types not listed in this OCP doc chart? 

Yes, as you see https://github.com/openshift/openshift-docs/pull/49799, it called **_tested_**, actually more types are supported.